truffleruby/src/main/java/org/truffleruby/cext/CExtNodes.java

```diff
@@ -250,6 +251,56 @@ public Object ulong2num(long num) {
 
     }
 
+    @CoreMethod(names = "rb_big_pack", isModuleFunction = true, required = 2)
+    public abstract static class BIGPACKNode extends CoreMethodArrayArgumentsNode {
```

Comment:
All the existing nodes previous to this PR seem to follow CamelCase from a quick look.